print-udp.c

```diff
@@ -646,6 +646,8 @@ udp_print(netdissect_options *ndo, register const u_char *bp, u_int length,
 			geneve_print(ndo, (const u_char *)(up + 1), length);
 		else if (IS_SRC_OR_DST_PORT(LISP_CONTROL_PORT))
 			lisp_print(ndo, (const u_char *)(up + 1), length);
+		else if (dport == GUE_PORT)
```

Comment:
Hrm, I'll have to look more closely for that, I forgot about that case. Could that be metadata/OOB-only with no actual IP payload inside?